src/nami.js

```diff
@@ -239,17 +240,17 @@ Nami.prototype.close = function () {
  * Opens the connection to AMI.
  * @returns void
  */
-Nami.prototype.open = function () {
+Nami.prototype.open = function (opts = {}) {
```

Comment:
I think this should be specified inside the config object passed to the Nami constructor